src/server/responses.py

```diff
@@ -80,3 +82,13 @@ def internal_database_error(_message: Message) -> Response:
             {"message": "Internal server error"},
             500,
         )
+
+    @staticmethod
+    def chart(filtered_list: list[dict]) -> Response:
+        """200: success on chart endpoint."""
+        return make_response(json.dumps(filtered_list).encode("utf-8"))
```

Comment:
`json.dumps` is not necessary here, I believe that `make_response` will jsonify it anyway ([docs](https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.make_response)).

Reply:
Done